File: demo-source/乘法口诀表.js
for (let row = 1; row <= 9; row += 1) {
  const cells = [];
  for (let column = 1; column <= row; column += 1) {
    cells.push(`${column}×${row}=${String(column * row).padStart(2, " ")}`);
  }
  console.log(cells.join("\t"));
}

File: demo-source/乘法口诀表.java
public class 乘法口诀表 {
    public static void main(String[] args) {
        for (int row = 1; row <= 9; row++) {
            for (int column = 1; column <= row; column++) {
                System.out.printf("%d×%d=%2d\t", column, row, column * row);
            }
            System.out.println();
        }
    }
}
